### ISSUE TITLE
iPhone 15 Pro Max App Store Connect resolution for videos is wrong

### ISSUE THREAD

**Issue by AvdLee** [AvdLee/RocketSimApp#476]
Reported by Alexander via mail.

**Comment by AvdLee** [comment]
@alexsereno these are the requirements taken from [here](https://help.apple.com/app-store-connect/?lang=en/#/dev4e413fcb8):

<img width="871" alt="CleanShot 2024-03-25 at 15 12 05@2x" src="https://github.com/AvdLee/RocketSimApp/assets/4329185/45e253f8-4c52-4c97-b6a6-1458eed17f96">

I took a screenshot and this is the outcome resolution:

![CleanShot 2024-03-25 at 15 12 29@2x](https://github.com/AvdLee/RocketSimApp/assets/4329185/4663015a-fba4-47ee-add3-b3471551a673)


In other words, I'm closing this issue as cannot reproduce.

Feel free to reopen with more details.

**Comment by alexsereno** [comment]
Hey @AvdLee sorry to reopen but screenshots have a different resolution requirement than the App Preview (videos.)
I made the same mistake at first! My original bug report was "RocketSim’s iPhone 15 Pro Max app preview recordings aren’t optimized correctly for App Store connect."

For uploaded screenshots, the 1290x2796 requirement is correct. But check out the app preview specifications and pay attention to the resolution requirement, new uploads keep getting rejected.

Screenshot from [here](https://developer.apple.com/help/app-store-connect/reference/app-preview-specifications):

<img width="869" alt="image" src="https://github.com/AvdLee/RocketSimApp/assets/2569805/077d00ca-1e77-46f4-97a0-32a414125a01">

The resolution requirement for the previews is as follows:

886 x 1920 pixels

(portrait)

1920 x 886 pixels

(landscape)

Any RocketSim preview I've made comes out at 1290 × 2796.

Record w/ optimize for app store selected:
<img width="638" alt="image" src="https://github.com/AvdLee/RocketSimApp/assets/2569805/dd5d0278-31cd-4d23-8488-e728e89e0b8f">

Output resolution is not acceptable:
<img width="512" alt="Screenshot 2024-03-25 at 9 38 07 AM" src="https://github.com/AvdLee/RocketSimApp/assets/2569805/7a328300-7471-4886-ae2d-a5941510713c">

**Comment by alexsereno** [comment]
While I don't have permissions to reopen the issue I want to follow up to note that this issue is present on iPhone 15 Pro (non-max) sims as well, with an output resolution of 1178 × 2556

<img width="367" alt="image" src="https://github.com/AvdLee/RocketSimApp/assets/2569805/3b362197-3e16-47a1-a410-79652a48bea7">

**Comment by AvdLee** [comment]
@alexsereno thanks for letting me know! I didn't know this was about videos. I'll reopen!
